dashboard/assets/js/containers/product-info-tab.js

```diff
@@ -200,6 +200,64 @@ export class ProductInfo extends Component {
     );
   }
 
+  codeClimateProjectURL(githubURL) {
+    const a = document.createElement('a');
+    a.href = githubURL;
+    return 'https://codeclimate.com/github' + a.pathname;
+  }
+
+  handleImageError(event) {
+    event.currentTarget.style.display = 'none';
```

Comment:
the react way is to emit a state update and leave it for react to re-render the view. this is probably more idiomatic.

```
class ImageDisappearOnError extends Component {

  constructor(props) {
    super(props);
    this.state = { hasError: false };
  }

  handleError() {
    this.setState({ hasError: true });
  }

  render() {
    if (this.state.hasError) {
      return null;
    };
    return <img onError={() => this.handleError()} src={ this.props.src } />;
  }
}
```

example usage:

```
<ImageDisappearOnError 
  src={ this.codeClimateProjectURL(link.url) + "/badges/gpa.svg" }/>

```